install_config/upgrading/automated_upgrades.adoc

```diff
@@ -526,15 +526,32 @@ following required variable within the `[OSEv3:vars]` section:
 ----
 [OSEv3:vars]
 
-openshift_logging_install_logging=true <1>
-openshift_logging_image_version=<tag> <2>
+openshift_hosted_logging_deploy=true <1>
+openshift_hosted_logging_deployer_prefix=registry.example.com:8888/openshift3/ <2>
+openshift_hosted_logging_deployer_version=3.6.0 <3>
 ----
 <1> Enables the ability to upgrade the logging stack.
-<2> Replace `<tag>` with `{latest-int-tag}` for the latest version.
+<2> Replace `registry.example.com:8888/openshift3/` with your desired prefix. 
+<3> Replace with the desired image version, or with `{latest-int-tag}` for the latest version.
 
-. Add any other `openshift_logging_*` variables that you want to specify to
-override the defaults, as described in
-xref:../../install_config/aggregate_logging.adoc#aggregate-logging-ansible-variables[Specifying Logging Ansible Variables].
+. Add any other `openshift_logging_*` or `openshift_hosted_logging_*` variables
+that you want to specify to override the defaults, as described in
```

Comment:
remove "to specify"? `that you want to override the defaults` _sounds_ cleaner and the `Add any` should cover the `to specify`.